mail-client/mutt-wizard/mutt-wizard-20200209.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A system for automatically configuring mutt and isync"
+HOMEPAGE="https://github.com/LukeSmithxyz/mutt-wizard"
+
+COMMIT=d8f57b57f6e4f7d07969a26775686b45b0e2b565
+SRC_URI="https://github.com/LukeSmithxyz/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+notmuch links +w3m lynx +libnotify +abook +gpg +cron +imagemagick"
```

Comment:
This looks like a series of shell scripts with no configurability during build-time, hence using USE flags here is banned.
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001

Reply:
I have removed the unneeded flags